sqlite_fdw.c

```diff
@@ -5738,6 +5739,8 @@ sqlite_affinity_eqv_to_pgtype(Oid type)
 	}
 }
 
+static const char *azType[] = { "?", "integer", "real", "text", "blob", "null" };
```

Comment:
Const should be declared at the beginning of the file. Could you move this array to there?

Reply:
Moved after declaration of functions. Please verify @son-phamngoc !

Reply:
Thanks for fixing. I confirmed. Please help me to resolve this comment.